cmd/util/cmd/epochs/cmd/recover.go

```diff
@@ -20,22 +21,20 @@ import (
 // identities, generates the cluster QC's and retrieves the DKG key vector of the last successful epoch.
```

Comment:
```suggestion
// identities, generates the cluster QC's and retrieves the DKG key vector of the last successful epoch.
// This recovery process has some constraints:
//  - The RecoveryEpoch must have exactly the same consensus committee as participated in the most recent successful DKG.
//  - The RecoveryEpoch must contain enough "internal" collection nodes so that all clusters contain a supermajority of "internal" collection nodes (same constraint as sporks)
```